ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -415,18 +506,42 @@ Section 001`
                 <Typography>Review your CSV file</Typography>
                 <CloudDoneIcon className={confirmationClasses.dialogIcon} fontSize='large'/>
                 <Typography>Your file is valid!  If this looks correct proceed with download</Typography>
-                <Button variant="outlined" onClick={(e) => resetPageState()}>Do Something</Button>
+                <Button variant="outlined" onClick={(e) => resetPageState()}>Cancel</Button>
+                <Button variant="outlined" onClick={async () => await doSaveCanvasSectionData()}>Sumbit</Button>
               </Paper>
             </Grid>
           </Box>
         </Grid>
+        <Backdrop className={classes.backdrop} open={isSaveCanvasSectionDataLoading}>
+        <Grid container>
+          <Grid item xs={12}>
+            <CircularProgress color="inherit" />
+          </Grid>
+          <Grid item xs={12}>
+          {renderLoadingText()}
+          </Grid>
+        </Grid>
+      </Backdrop>
       </div>)
   }
 
   const sectionNamesToSection = (sectionNames: string[]): Section[] => {
     return sectionNames.map((name, i) => { return { rowNumber: i + 1, sectionName: name } })
   }
 
+  const renderSuccess = (): JSX.Element => {
+    return (
+      <Card className={successClasses.card} variant="outlined">
+        <CardContent className={successClasses.cardContent}>
+          <CheckCircle className={successClasses.icon} fontSize='large'/>
+          <Typography>New sections have been added!</Typography>
+        </CardContent>
+        <CardActions className={successClasses.cardFooter}>
+          See your sections in the Canvas Setting
```

Comment:
We need to provide a link to the page setting `/courses/:course_id/settings#tab-sections` to see if sections has been added or not.

Reply:
This requires additional functionality that were going to handle in a separate issue.  The canvas url needs to be loaded into globals

Reply:
The globals already is sending the `course_id`, why can't we make the URL like `/courses/403334/setting#tab-sections`?

global call response
`{"environment":"development","userLoginId":"dddd","course":{"id":403334,"roles":["TeacherEnrollment","Account Admin"]}}`

Reply:
We know the ending of the url but not the environment dependent beginning

Reply:
relative URL's won't work in this case? just like `<a href = '/course/40404/setting#tab-section'` or using a link Material UI component?